src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplateBase/help-extraDockerLabelsString.html

```diff
@@ -0,0 +1,14 @@
+<div>
+    A list of new line separated <em>docker</em> labels to be set on the container.
+    Specified in the form "label_name=value".
+
+    This has no effect on Jenkins functionality, and is not related to the labels
+    used by Jenkins to map jobs to slave nodes. These labels are metadata attached
+    to the docker container itself (again, not the jenkins slave), and which can
+    typically be read using the <pre>docker inspect</pre> command.
+
+    This can be useful when using a docker cluster (like docker swarm) to pass
+    information to the scheduler, in conjunction with constraints; or to let
+    other services (portainer, prometheus...) know how they should categorize or
+    otherwise deal with this specific container.
+</div>
```

Comment:
You've got a typo "spaces are the beginning" should be "spaces at the beginning".

Also, this file is HTML; it is not plain-text.  Line-breaks in this file will not be preserved when it's displayed in the Jenkins WebUI.
You need to add formatting to it, otherwise it'll look awful.
e.g. Wrap each paragraph in `<p>`...`</p>` and the bullet points at the end should be done as
```
Note:<ul>
<li>spaces at the beginning and end of the label name and value will be removed</li>
<li>invalid lines will be ignored</li>
</ul>
```